packages/@expo/cli/src/start/server/metro/inspector-proxy/messages/__tests__/VscodeCompat.test.ts

```diff
@@ -0,0 +1,69 @@
+import { RuntimeGetProperties, VscodeCompatHandler } from '../VscodeCompat';
+
+describe(VscodeCompatHandler, () => {
```

Comment:
top-level describe isn't needed and adds unneeded complexity to mock resetting.

Reply:
Sure, I can remove it.

Reply:
Let me remove it after I merged a few follow-up PRs like: #21566, else I'll be rebasing into oblivion

Reply:
done